README.md

```diff
@@ -17,18 +17,25 @@ Ensure you have the following installed:
 
 Now we can download Drupal 8 and import our Drupal 7 assets.
 
- 1. Copy your Drupal 7 database dump to ./imports/data and site files to ./imports/files (Note that you should copy the 'sites' directory into here, so that the path './imports/files/sites/default/files/articles' exists)
- 2. Make a copy of .lando.example.yml naming it .lando.local.yml and edit with your own unique project name.
- 3. Create config/.env with any local changes you require: `cp config/.env.sample config/.env`. You'll need to populate the sensitive values by consulting a member of the project team or having access to the hosting environment.
- 4. Run *'lando start'*
- 5. Run *'lando db-import -h drupal7db ./imports/data/[SQL DUMP FILENAME].sql'*
- 6. Open your lando site url (displayed at the end of 'lando start', or use *'lando info'*)
- 7. Proceed with the Drupal 8 installation.
- 8. On the 'Select an installation profile', check 'Use existing configuration' and continue.
+ 1. Make a copy of .lando.example.yml naming it .lando.local.yml and edit with your own unique project name.
+ 2. Create config/.env with any local changes you require: `cp config/.env.sample config/.env`. 
+	As a minimum you'll need to set the 'HASH_SALT' to a random string and set CONFIG_SYNC_DIRECTORY to '../config/sync'
+ 3. Run *'lando start'*
+ 4. From the /drupal directory run 'lando composer install'
+ 5. Open your lando site url (displayed at the end of 'lando start', or use *'lando info'*)
+ 6. Proceed with the Drupal 9 installation.
```

Comment:
Might be good to separate into something like: if you have a db dump, you can now import it. If you're installing from scratch then follow the installer form or use drush.

You might also find with the Drupal 9 recipe that there's no global drush command, so that'll either need adding to the tooling section of Lando, or you need to use `vendor/bin/drush` each time. Might just be an issue on platform.sh... I forget :)